docs/articles/basics.md

```diff
@@ -76,13 +76,17 @@ then it is trivial for someone to gain root access to the host where the
 daemon is running.
 
 Similarly, the Docker client can use `-H` to connect to a custom port.
+The Docker client will default to connecting to `unix:///var/run/docker.sock`
+on Linux, and `tcp://127.0.0.1:2376` on Windows.
 
 `-H` accepts host and port assignment in the following format:
 
-    tcp://[host][:port][path] or unix://path
+    tcp://[host:][port][path] or unix://path
```

Comment:
> What on earth does [path] mean in terms of a daemon listening? At least on Windows (and my admittedly 25-years-ago-since-I-last-used-or-programmed-Unix knowledge), AFAIK, you can only bind/listen/accept in TCP end-point terminology (OSI Layer 4) on an IP address/port pair, not a trio of address/port/path. A path is only relevant higher up in the OSI stack such as an HTTP server. Is there something different on Unix that isn't (to the best of my knowledge) relevant on Windows???

For the `path` portion, it comes from here : #14304 (and the issue :  #14270)